modules/geo/src/main/java/org/opensearch/geo/search/aggregations/bucket/geogrid/InternalGeoGrid.java

```diff
@@ -54,7 +54,7 @@
  * All geo-grid hash-encoding in a grid are of the same precision and held internally as a single long
  * for efficiency's sake.
  *
- * @opensearch.internal
+ * @opensearch.api
```

Comment:
I am not sure if we want to add `api` here. @nknize can get some suggestion from your side.